scripts/common_cmdline.sh

```diff
@@ -64,7 +64,8 @@ function compileFull()
     local exit_code=$?
     local errors; errors=$(grep -v -E \
         -e 'Warning: This is a pre-release compiler version|Warning: Experimental features are turned on|pragma experimental ABIEncoderV2|^ +--> |^ +\||^[0-9]+ +\| ' \
-        -e 'Warning: Yul is still experimental. Please use the output with care.' < "$stderr_path")
+        -e 'Warning: Yul is still experimental. Please use the output with care.' \
+        -e '^No text representation found.$' < "$stderr_path")
```

Comment:
so you basically just add it to the ignore-list?

Reply:
This does not ignore the test itself. Just strips this particular piece of text. If it stops being compilable we will still get an error.

Reply:
Yep, as discussed in the chat, it's a valid result of the test. The function in the code is never called so it's never generated.